### ISSUE TITLE
Implement easy toggling between only preview and only tex

### ISSUE THREAD

**Issue by johnjeng** [sagemathinc/cocalc#1372]
A common usage pattern for students is split screen homework on one side in a browser window and their tex in the other. On the vast majority of screens, vertically split latex is useless in this case. 

Should wait on LaTeX editor rewrite for this since it's a relatively low priority (dragging the slider over suffices for the most part).

**Comment by williamstein** [comment]
I have absolutely no idea what you are proposing here.

**Comment by johnjeng** [comment]
The following setup is extremely common. But note how the latex editor split is not ideal.
![split](https://cloud.githubusercontent.com/assets/618575/21164630/3e5031d8-c14f-11e6-9c4b-712899ffc901.png)

Really there's just enough for one view or the other.
![pdf](https://cloud.githubusercontent.com/assets/618575/21164632/3ff7746a-c14f-11e6-8b42-bd24e9fa7608.png)
![tex](https://cloud.githubusercontent.com/assets/618575/21164636/40f0e6e4-c14f-11e6-9905-0378784bd441.png)


It's more ideal to either split horizontally or have a quicker way than dragging the bar. Both might be a good idea. Anyways, just something my friends have run into.

**Comment by williamstein** [comment]
OK; now I see. Try out share latex -- they have such a toggle, I think, as a button on the bar itself.

**Comment by williamstein** [comment]
our frame editor easily lets you do this sort of thing now.
